.github/workflows/ci.yml

```diff
@@ -50,3 +50,11 @@ jobs:
         image: scoriacorp/tls_memcached:latest
         ports:
         - 11211/tcp
+
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: psf/black@stable
+        with:
+          version: "21.5b1"
```

Comment:
Given that black is already configured in `tox.ini`, you could drop this step entirely and just update the existing lint step above:

```diff
@@ -29,7 +29,7 @@ jobs:
       if: matrix.python-version == '3.9'
       run: |
         pip install tox tox-gh-actions
-        tox -e flake8
+        tox -e black,flake8
     - name: Disable IPv6 localhost
       run: |
         sudo sed -i '/::1/d' /etc/hosts
```